orte/mca/oob/tcp/oob_tcp_listener.c

```diff
@@ -909,7 +936,9 @@ static void connection_event_handler(int incoming_sd, short flags, void* cbdata)
 static void tcp_ev_cons(mca_oob_tcp_listener_t* event)
 {
     event->ev_active = false;
+    event->tcp6 = false;
     event->sd = -1;
+    event->port = -1;
```

Comment:
The port is a `uint16_t` -- this assignment may well generate a compiler warning...?